build.gradle

```diff
@@ -54,24 +55,4 @@ subprojects {
     installDist {
         duplicatesStrategy DuplicatesStrategy.EXCLUDE
     }
-
-    task sourcesJar(type: Jar) {
-        classifier = 'sources'
-        from sourceSets.main.allSource
-    }
-
-    task javadocJar(type: Jar) {
-        classifier = 'javadoc'
-        from javadoc
-    }
-
-    task testJar(type: Jar) {
```

Comment:
I'm just curious. Any reason you decided not to publish `test` and `testSources`?

Reply:
Including `test` and `testSources` jars to a release artifact seems to be a bit tricky in the `maven-publish` plugin. And I rethought why we needed to publish them, and I thought we didn't need to do that, at least for now.

What do you think?

Reply:
I think it's fine not to have them since we don't really use them.
Thank you!